app/main/helpers/hash_email.py

```diff
@@ -0,0 +1,9 @@
+import hashlib
+import base64
+
+
+def hash_email(email):
```

Comment:
I think we have a copy of this function in utils (https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/email/helpers.py). Can we use that?